src/me/mrCookieSlime/Slimefun/Commands/SlimefunCommand.java

```diff
@@ -184,23 +186,41 @@ else if (args[0].equalsIgnoreCase("versions")) {
 				}
 			}
 			else if (args[0].equalsIgnoreCase("give")) {
-				if (args.length == 3) {
-					if (sender.hasPermission("slimefun.cheat.items") || !(sender instanceof Player)) {
+			    if (sender.hasPermission("slimefun.cheat.items") || !(sender instanceof Player)) {
+			        if (args.length == 3) {
 						if (Players.isOnline(args[1])) {
 							if (Slimefun.listIDs().contains(args[2].toUpperCase())) {
-								if (Players.isOnline(args[1])) {
-									Messages.local.sendTranslation(Bukkit.getPlayer(args[1]), "messages.given-item", true,new Variable("%item%", SlimefunItem.getByName(args[2].toUpperCase()).getItem().getItemMeta().getDisplayName()));
-									Bukkit.getPlayer(args[1]).getInventory().addItem(SlimefunItem.getByName(args[2].toUpperCase()).getItem());
-									Messages.local.sendTranslation(sender, "messages.give-item", true, new Variable("%player%", args[1]), new Variable("%item%", SlimefunItem.getByName(args[2].toUpperCase()).getItem().getItemMeta().getDisplayName()));
-								}
+								Messages.local.sendTranslation(Bukkit.getPlayer(args[1]), "messages.given-item", true, new Variable("%item%", SlimefunItem.getByName(args[2].toUpperCase()).getItem().getItemMeta().getDisplayName()), new Variable("%amount%", "1"));
+								Bukkit.getPlayer(args[1]).getInventory().addItem(SlimefunItem.getByName(args[2].toUpperCase()).getItem());
+								Messages.local.sendTranslation(sender, "messages.give-item", true, new Variable("%player%", args[1]), new Variable("%item%", SlimefunItem.getByName(args[2].toUpperCase()).getItem().getItemMeta().getDisplayName()), new Variable("%amount%", "1"));
 							}
 							else Messages.local.sendTranslation(sender, "messages.not-valid-item", true, new Variable("%item%", args[2]));
 						}
 						else Messages.local.sendTranslation(sender, "messages.not-online", true, new Variable("%player%", args[1]));
 					}
-					else Messages.local.sendTranslation(sender, "messages.no-permission", true);
+			        else if (args.length == 4){
+			            if (Players.isOnline(args[1])) {
+                            if (Slimefun.listIDs().contains(args[2].toUpperCase())) {
+                                 try {
+                                     int amount = Integer.parseInt(args[3]);
+                                     
+                                     if (amount > 0) {
```

Comment:
With this, you can use anything above zero for the amount, even items bigger than 64.
Do a for loop that deducts the max stack size of an item instead.